doc/backend_vars.asciidoc

```diff
@@ -336,6 +336,8 @@ GENERAL_HW_POWERON_CMD;string;;Shell Command to power on the SUT (in CMD_DIR)
 GENERAL_HW_POWERON_ARGS;string;;Arguments to pass GENERAL_HW_POWERON_CMD Shell script
 GENERAL_HW_POWEROFF_CMD;string;;Shell Command to power off the SUT (in CMD_DIR)
 GENERAL_HW_POWEROFF_ARGS;string;;Arguments to pass GENERAL_HW_POWEROFF_CMD Shell script
+GENERAL_HW_IS_SHUTDOWN_CMD;string;;Shell Command to check if SUT is powered down (in CMD_DIR). Should return exit code 0 if it is, or 1 otherwise.
```

Comment:
```suggestion
GENERAL_HW_IS_SHUTDOWN_CMD;string;;Shell command to check if SUT is powered down (in CMD_DIR). Should return exit code 0 if it is, or 1 otherwise.
```
Although I'm aware that the casing is also wrong in the line you copied this from. So you might also just leave it or change all of those lines.